src/znc.cpp

```diff
@@ -1717,9 +1717,70 @@ bool CZNC::AddListener(unsigned short uPort, const CString& sBindHost,
     return true;
 }
 
+bool CZNC::AddListener(const CString& sPath, const CString& sURIPrefixRaw,
+                       bool bSSL, CListener::EAcceptType eAccept,
+                       CString& sError) {
+    CUtils::PrintAction("Binding to path [" + sPath + "]");
+
+#ifndef HAVE_LIBSSL
+    if (bSSL) {
+        sError = "SSL is not enabled";
+        CUtils::PrintStatus(false, sError);
+        return false;
+    }
+#else
+    CString sPemFile = GetPemLocation();
```

Comment:
SSL code should be shared between TCP and Unix sockets...

Reply:
The arguments to PrintAction like 1749 and 1683 are diffrent.